core/templates/dev/head/components/fuzzy_rule_editor_directive.html

```diff
@@ -0,0 +1,2 @@
+<input type="hidden"/>
+<button type="button" class="btn btn-danger" ng-click="removeEntry()">Remove this input</button>
```

Comment:
Hi @prasanna08, I'm sorry but my AngularJS knowledge is pretty limited so I'm not sure if I'm the right person to review it. @AllanYangZhou is also busy with his exams right now. @seanlip thanks for your review!